src/main.rs

```diff
@@ -362,6 +362,14 @@ fn create_app(default_vcpus: String, default_memory: String, default_rng: String
                 .num_args(1)
                 .value_parser(["true", "false", "log"])
                 .default_value("true"),
+        )
+        .arg(
+            Arg::new("tpm")
```

Comment:
Ah here I can see it doesn't match the documentation, but it matches my expectations.